packages/wdio-config/src/node/ConfigParser.ts

```diff
@@ -199,6 +199,13 @@ export default class ConfigParser {
             this._config.cucumberFeaturesWithLineNumbers = Array.isArray(this._config.spec) ? [...new Set(this._config.spec)] : [this._config.spec]
         }
 
+        /**
+         * Make glob-pattern from CLI --spec arg if it is only a keyword
+         */
+        if (spec.length > 0 && !this.cliSpecArgContainsPathstoSpecs(spec) && !this.isGlobPattern(spec[0])){
+            spec[0] = this.addGlobPatternToSpecCliArg(spec[0])
```

Comment:
Is this really needed? Can we just do this later in `filterSpecsFromConfigFileByKeyword`?